css/styles.css

```diff
@@ -56,6 +56,90 @@ a {
   display: none;
 }
 
+.popup {
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: white;
+  border: 15px solid #c1c7d0;
+}
+
+.xsign {
+  position: absolute;
+  top: 0;
+  margin-left: 298px;
+  padding-top: 50px;
+  color: #fff;
+  z-index: 1;
+}
+
+.pop-img {
+  width: 370px;
+  position: relative;
+  height: 320px;
+  padding-right: 35px;
+}
+
+.phead {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 260px;
+  margin-left: 10px;
+}
+
```

Comment:
- [OPTIONAL] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 


- Please blur the sub background when the popup is open, it will make the popup more like Figma 👍🏿
- It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 
- Make sure the max width of your popup windows is 920px as shown on the Figma design

Mobile  | Desktop
:---:|:---:
![181](https://user-images.githubusercontent.com/79058364/189106792-88e6a0ec-99f7-4c4e-923e-759a412258dc.png) | ![18](https://user-images.githubusercontent.com/79058364/189106145-f947cebe-3a5a-45a8-9851-ada6c6544c46.png) 


- [OPTIONAL] I don't know if I'm the only one who can say that your design is far from the original model, in fact I can't mark it as required because the previous reviewer didn't say anything about it, however,  I strongly suggest you improve your app design as you can see here 👇🏻

![Screenshot (1)](https://user-images.githubusercontent.com/79058364/209351312-34c1e2ff-dcdb-4dd0-b944-f2fcba21fe4f.png)